packages/adapters-library/src/adapters/iziswap/tests/testCases.ts

```diff
@@ -45,18 +47,6 @@ export const testCases: TestCase[] = [
       tokenId: '22264',
     },
   },
-  {
-    chainId: Chain.Bsc,
-    method: 'withdrawals',
-    input: {
-      userAddress: '0x4F21a0Af178d9EB793c2D94d9d88910e974a98fD',
-      fromBlock: 34127856,
-      toBlock: 34127858,
-      protocolTokenAddress: '0xBF55ef05412f1528DbD96ED9E7181f87d8C9F453',
-      productId: 'pool',
-      tokenId: '85',
-    },
-  },
```

Comment:
No withdrawals for iziswap on BSC recent enough for the request to work with our rpc provider.